src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
+date: 2020-12-09
+hero: hero.jpg
```

Comment:
Is this how you want it to appear on web.dev/blog? Except for the `draft` thing (you can ignore that)

![image](https://user-images.githubusercontent.com/4713486/101400870-5bb08400-3886-11eb-86fa-e34dd40bdbef.png)